src/engraving/dom/cmd.cpp

```diff
@@ -3304,18 +3304,13 @@ void Score::cmdIncDecDuration(int nSteps, bool stepDotted)
         pasteStaff(e, selection().startSegment(), selection().staffStart(), scale);
         return;
     }
-    if (selection().isList() && selection().elements().size() > 1) {
-        // List - act as if pressing duration toggle (distinct from range based Half/Double
-        TDuration newDuration(stepDotted
-                              ? m_is.duration().shiftRetainDots(nSteps, stepDotted)
-                              : m_is.duration().shift(nSteps));
-        m_is.duration().shiftRetainDots(nSteps, stepDotted);
-        m_is.setDuration(newDuration);
-        std::set crs = getSelectedChordRests();
+    if (selection().isList()) {
         for (auto cr : getSelectedChordRests()) {
+            TDuration newDuration(stepDotted
+                                  ? cr->durationType().fraction() * Fraction(3, (3 + nSteps))
+                                  : cr->durationType().fraction()* Fraction(3 - nSteps, 3 + nSteps));
```

Comment:
Serious?? Code style check wants it this way, no space between `fraction()`and `*` ?!?